src/main/java/org/folio/rest/support/StatusUpdatedDateGenerator.java

```diff
@@ -0,0 +1,13 @@
+package org.folio.rest.support;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public final class StatusUpdatedDateGenerator {
+
+    public static String generateStatusUpdatedDate() {
+      SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
+      return sdf.format(new Date());
+    } 
+    
```

Comment:
Personally, I would remove the blank lines around this method, I don't think they add to the readability of the code.

```suggestion
    public static String generateStatusUpdatedDate() {
      SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
      return sdf.format(new Date());
    } 
```